vita3k/host/src/load_self.cpp

```diff
@@ -342,7 +342,12 @@ SceUID load_self(Ptr<const void> &entry_point, KernelState &kernel, MemState &me
         LOG_CRITICAL("SELF {} header type {} is not supported.", self_path, self_header.header_type);
         return -1;
     }
-
+    
+    if (self_path == "app0:sce_module/steroid.suprx") {
+        app::error_dialog("You're trying to load a vitamin dump. It is not supported.", nullptr);
```

Comment:
Seems unnecessary to place this fix here.

Why not in `interface.cpp` instead?